### PR TITLE
feat(api-client): Remove last bits of contact import

### PR DESCRIPTION
**Pull request by ffflorian**
## Pull Request Checklist

- [ ] My code is covered by tests
- [x] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/api-client/src/user/data/UserPropertiesSetData.ts [hunk 1]
```diff
@@ -32,10 +32,6 @@ export enum NotificationPreference {
 
 export interface WebappProperties {
   [property_key: string]: any;
-  contact_import: {
-    google?: number;
-    macos?: number;
-  };
   enable_debugging: boolean;
```
Comment (ffflorian) [inline packages/api-client/src/user/data/UserPropertiesSetData.ts:35]: I am wondering if these properties should be optional?

Comment (bennyn) [reply packages/api-client/src/user/data/UserPropertiesSetData.ts:35]: Let's remove them.

Comment (ffflorian) [reply packages/api-client/src/user/data/UserPropertiesSetData.ts:35]: I meant the other properties like `enable_debugging`.

Comment (bennyn) [reply packages/api-client/src/user/data/UserPropertiesSetData.ts:35]: Does it cause any problems having them to be mandatory?

Comment (ffflorian) [reply packages/api-client/src/user/data/UserPropertiesSetData.ts:35]: For now it doesn't :)

**Pull request merged**